roles/common/tasks/password-policy.yml

```diff
@@ -1,8 +1,16 @@
----
+--
```

Comment:
Hah, this went from 4 dashes, to 2. I think the standard is 3, but this may work anyway.